en/docs/install-and-setup/upgrading-wso2-api-manager/upgrading-from-260-to-320.md

```diff
@@ -1576,6 +1580,10 @@ Follow the instructions below to move all the existing API Manager configuration
         ALTER TABLE AM_API_RATINGS
             ADD RATING_ID VARCHAR(255) DEFAULT (SYS_GUID()) NOT NULL
         /
+        
+        ALTER TABLE AM_API_RATINGS
+            ADD CONSTRAINT add_pk PRIMARY KEY (RATING_ID)
```

Comment:
Let's add a different name to this constraint. This should be unique. (add_pk)

Reply:
Changed to add_pk_ratings.